playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2

```diff
@@ -16,6 +16,7 @@ disabled    : "{{ (inventory_hostname != groups['mons'][0] or check_name | match
 details     :
     file    : run_plugin_in_venv.sh
     args    : {{ ceph_args }}
+    timeout : {{ (maas_check_timeout_override[label] | default(maas_check_timeout) * 1000) }}
```

Comment:
Lets not change all templates and much rather calculate it via a default variable and convert maas_check_timeout into maas_check_timeout * 1000 if we detects the input is seconds as example (ie the timeout is only 3 digits long etc)

Reply:
The problem is that the check-level still utilizes seconds, but doesn't actually do anything for `agent.plugin` checks. These check types require a millisecond declaration:

https://developer.rackspace.com/docs/rackspace-monitoring/v1/tech-ref-info/check-type-reference/#agent-plugin